src/model/location-schema/location.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const mongoose = require('mongoose');
+
+const locationSchema = mongoose.Schema({
+  // !Tom - adding the required line in case we want to make location required later
+  // !Tom - locationName is how the customer would name their locations, not necessarily
+  //        how they are sorted under the hood. Still figuring out how to do that on our end
+  locationName: {
```

Comment:
Hey @tnorth93 , looks good. Excited to write some routes and get this tested. I would say changing this to 'location' would be better. We know it will have a name. What are your thoughts?

Reply:
Opps! I misread this! locationName is needed! : )